README.md

```diff
@@ -126,10 +125,17 @@ documentation](https://prometheus.io/docs).
 
 ### Scrape meta data
 
-There are two metrics providing data about the scrape itself:
-
- - `ipmi_up` is `1` if all data could successfully be retrieved from the remote
-   host, `0` otherwise
+There are four metrics providing data about the scrape itself:
+
+ - `ipmi_up{collector="ipmi"}` is `1` if `ipmimonitoring` data could
+   successfully be retrieved from the remote host, `0` otherwise. If it is `0`,
```

Comment:
I would simplify this documentation by just talking about `ipmi_up{collector="XXX"`, and having a table of collectors that are available. This avoids too much copy-pasta. :-)

Reply:
Good point, will do